backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -585,15 +588,23 @@ export class UserResolver {
     })
 
     const loginUserBackupRepository = await getRepository(LoginUserBackup)
-    const loginUserBackup = await loginUserBackupRepository
-      .findOneOrFail({ userId: loginUser.id })
-      .catch(() => {
-        throw new Error('Could not find corresponding BackupUser')
-      })
+    let loginUserBackup = await loginUserBackupRepository.findOne({ userId: loginUser.id })
+
+    // Generate Passphrase if needed
+    if (!loginUserBackup) {
+      const passphrase = PassphraseGenerate()
+      loginUserBackup = new LoginUserBackup()
+      loginUserBackup.userId = loginUser.id
+      loginUserBackup.passphrase = passphrase.join(' ') // login server saves trailing space
+      loginUserBackup.mnemonicType = 2 // ServerConfig::MNEMONIC_BIP0039_SORTED_ORDER;
+      loginUserBackupRepository.save(loginUserBackup)
+    }
```

Comment:
I think would be prettier like this:
```suggestion
    const loginUserBackup = await loginUserBackupRepository
      .findOneOrFail({ userId: loginUser.id })
      .catch(() => {
        const passphrase = PassphraseGenerate()
        loginUserBackup = new LoginUserBackup()
        loginUserBackup.userId = loginUser.id
        loginUserBackup.passphrase = passphrase.join(' ') // login server saves trailing space
        loginUserBackup.mnemonicType = 2 // ServerConfig::MNEMONIC_BIP0039_SORTED_ORDER;
        loginUserBackupRepository.save(loginUserBackup)
      })
```

Reply:
I like to keep it like I have it, since its easier to read and understand. Hope thats ok.